engine/apps/schedules/tests/test_quality_score.py

```diff
@@ -190,7 +190,7 @@ def test_get_schedule_score_weekdays(
     assert response.json() == {
         "total_score": 86,
         "comments": [
-            {"type": "warning", "text": "Schedule has gaps (29% not covered)"},
+            {"type": "warning", "text": "Schedule has gaps (28% not covered)"},
```

Comment:
This value slightly changed because the (start, end) limits for the period taken into account has changed minimally (by ~1 second) when refactoring the backend implementation.